tests/README.md

```diff
@@ -1,3 +1,137 @@
+# Getting Started
+
+##  What are UI Tests?
+
+UI (User Interface) tests validate the visible and interactive parts of an application. They ensure that components render correctly, handle user interactions as expected, and provide a reliable user experience.
+
+### Why are UI tests important?
```

Comment:
Not sure if we need to have this in here as its quite generic stuff people should know, but also not feeling strongly to remove it

Reply:
I agree, I removed that part